Mathlib/Algebra/Module/LocalizedModule/IsLocalization.lean

```diff
@@ -28,18 +29,33 @@ theorem isLocalizedModule_iff_isLocalization {A Aₛ} [CommSemiring A] [Algebra
     simp [← IsScalarTower.algebraMap_apply, Submonoid.mk_smul, Algebra.smul_def, mul_comm]
   · congr!; simp_rw [Subtype.exists, Algebra.algebraMapSubmonoid]; simp [Algebra.smul_def]
 
-instance {A Aₛ} [CommSemiring A] [Algebra R A][CommSemiring Aₛ] [Algebra A Aₛ] [Algebra R Aₛ]
-    [IsScalarTower R A Aₛ] [h : IsLocalization (Algebra.algebraMapSubmonoid A S) Aₛ] :
+instance [IsLocalization (Algebra.algebraMapSubmonoid A S) Aₛ] :
     IsLocalizedModule S (IsScalarTower.toAlgHom R A Aₛ).toLinearMap :=
-  isLocalizedModule_iff_isLocalization.mpr h
+  isLocalizedModule_iff_isLocalization.mpr ‹_›
 
-lemma isLocalizedModule_iff_isLocalization' (R') [CommSemiring R'] [Algebra R R'] :
-    IsLocalizedModule S (Algebra.linearMap R R') ↔ IsLocalization S R' := by
-  convert isLocalizedModule_iff_isLocalization (S := S) (A := R) (Aₛ := R')
+variable (A)
+
+lemma isLocalizedModule_iff_isLocalization' :
```

Comment:
Can you add a docstring to this to remove the no lint while you are at it? Thanks!

Reply:
Done!